docs/topics/ref_resolved-issues-7-2-3.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+//docs/release_notes-7.2.0/master.adoc
+
+:_template-generated: 2024-12-04
+:_mod-docs-content-type: REFERENCE
+
+[id="resolved-issues-7-2-3_{context}"]
+= Fixed issues
+
+This section provides highlighted issues that have been resolved in {ProductFullName} 7.2.3.
```

Comment:
https://issues.redhat.com/browse/MTA-5406 is an additional bug fixed in MTA 7.2.3